README.md

```diff
@@ -30,6 +30,7 @@ Santiment API python client.
     - [Emerging Trends](#emerging-trends)
     - [Top Social Gainers Losers](#top-social-gainers-losers)
     - [Metric Complexity](#metric-complexity)
+    - [Available Since](#available-since)
```

Comment:
This should not be nested under `metrics` (as well as the metric complexity) I guess?